Sources/BraintreeCore/Public/BraintreeCore/BTAPIClient.h

```diff
@@ -91,6 +105,7 @@ typedef NS_ENUM(NSInteger, BTAPIClientErrorType) {
                       completion:(void(^)(NSArray <BTPaymentMethodNonce *> * _Nullable paymentMethodNonces, NSError * _Nullable error))completion;
 
 /**
+ :nodoc:
```

Comment:
Not sure I understand why `:nodoc:` is being added above the existing docs for these methods. Are they also only supposed to be used internally?

Reply:
Yeah, that was our thought. These methods default to using the REST API instead of GraphQL, and the methods that we're exposing let you choose which API to use. But really, all of these methods are meant to be used by other Braintree modules, not by merchants directly.

Reply:
Got it 👍